frontend/src/global.d.ts

```diff
@@ -1 +1,5 @@
 /// <reference types="@sveltejs/kit" />
+
+declare module '*.svg?component' {
```

Comment:
Please explain, does this allow us to import svg files as svelte components?

Reply:
Yes